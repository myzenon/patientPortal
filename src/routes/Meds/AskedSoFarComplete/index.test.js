import React from 'react'
import { AskedSoFarComplete } from './index'
import ReactGA from 'react-ga';

   // This should be part of your setup
   ReactGA.initialize('UA-121783543-1', {
       'debug': true
     });;

jest.mock("react-ga")

test('renders correctly', () => {
    const props = {
        patient: {
            ...mockObj.patient
        },
        cmr: {
            ...mockObj.cmr
        }
    }
    const wrapper = render(
        <AskedSoFarComplete { ...props } />
    )
    expect(toJson(wrapper)).toMatchSnapshot()
})