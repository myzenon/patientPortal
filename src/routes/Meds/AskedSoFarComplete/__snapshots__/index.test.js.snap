// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="container-fluid"
>
  <div
    class="content container"
  >
    <p>
      <strong>
        Rita
      </strong>
      , so far you have told as about
    </p>
    <p
      class="amountMedsText"
    >
      0  Medicines
    </p>
    <div
      class="medsListWrapper"
    >
      <ul />
    </div>
    <p
      class="buttonWrapper"
    >
      <a
        href="/meds-other-meds"
      >
        <button
          class="button btn btn-secondary"
        >
          GOT IT!
        </button>
      </a>
    </p>
  </div>
</div>
`;
